lib/DBIx/Class/ResultSource.pm

```diff
@@ -702,7 +702,7 @@ sub sequence {
 
 =over 4
 
-=item Arguments: $name?, \@colnames
+=item Arguments: $name?, \@colnames, \%extra?
 
```

Comment:
This is `add_unique_constraint` rather than `add_unique_constraints`. Shall I also change the plural version to accept hashrefs to follow suit?

This makes more sense, actually, now that I see what you're after.